awesomplete.js

```diff
@@ -95,7 +95,7 @@ var _ = function (input, o) {
 				li = li.parentNode;
 			}
 
-			if (li) {
+			if (li && evt.which === 1) {  // Only select on left click
```

Comment:
Given that `evt.which` is non-standard, I'd rather do `evt.which !== 2 && evt.which !== 3` or `[2,3].indexOf(evt.which) === -1` or something like that.